src/hosted-buttons/utils.js

```diff
@@ -98,7 +98,7 @@ export const getHostedButtonDetails: HostedButtonDetailsParams = async ({
       shape: getButtonVariable(variables, "shape"),
       color: getButtonVariable(variables, "color"),
       label: getButtonVariable(variables, "button_text"),
-      height: getButtonVariable(variables, "height"),
+      height: parseInt(getButtonVariable(variables, "height")) || undefined,
```

Comment:
Maybe this isn't possible from API, but `Boolean(parseInt('0'))` is `false`, so would end up `undefined` here

Reply:
Since `height` needs to be between 22-55 anyway, I don't think we have to worry about this 😄

Reply:
Though, I do have to add the radix